### PR TITLE
Remove admin mode

### PR DESCRIPTION
**Pull request by gewenyu99**
## What does this PR do?
Admin Mode is a concept that hasn't been applicable since we started shipping dedicated server vs client SDKs.

This PR removes references to Admin mode where applicable. Old API specs still reference the `docs/admin` page, so we can't remove the page. Instead, we did a short rewrite to explain client vs server SDKs to avoid further confusion.

Related to https://github.com/appwrite/homepage/pull/112

Screen Shot 2022-10-18 at 4 50 47 PM

---

Screen Shot 2022-10-18 at 4 55 42 PM

---

Screen Shot 2022-10-18 at 4 56 18 PM

## Test Plan

(Write your test plan here. If you changed any code, please provide us with clear instructions on how you verified your changes work.)

## Related PRs and Issues

(If this PR is related to any other PR or resolves any issue or related to any issue link all related PR and issues here.)

### Have you read the [Contributing Guidelines on issues](https://github.com/appwrite/appwrite/blob/master/CONTRIBUTING.md)?

(Write your answer here.)

### CODE REVIEW

File: app/views/docs/admin.phtml [hunk 1]
```diff
@@ -1,7 +1,9 @@
...
 
-<p>For security reasons, the admin mode only works in combination with an API key. You can create an API key from the Appwrite console, and you can choose what scopes of access you are willing to grant your SDK.</p>
+<p><a href="/docs/permissions"><i class="icon-angle-circled-right"></i>Learn more about permissions</a></p>
 
-<p>Please note: passing an API key from a client SDK is a <strong>major</strong> security issue. Use your API key only from a server-side integration and make sure you store them securely and privately. There are many sources online that advise what the best practices regarding the storing of secrets and API keys on your server are.</p>
+<p>When building server integrations with the Server SDKs, you operate in the scope of an API key instead of an account. API keys are created with scopes that specify the type of resources they can access and the type of operations they can perform on those resources. API keys bypass permissions and can access all resources belonging to any account.</p>
+
+<p><a href="/docs/keys"><i class="icon-angle-circled-right"></i>Learn more about API keys</a></p>
```
Comment (eldadfux) [inline app/views/docs/admin.phtml:9]: Something is off with the spacing of the icon, please see how we use it in other places. Maybe without the <p> tag

File: app/views/docs/permissions.phtml [hunk 2]
```diff
@@ -22,7 +22,7 @@
 
 <h2 id="server-integration"><a href="/docs/permissions#server-integration">Server Integration</a></h2>
 
-<p>A server or <a href="/docs/admin">admin integration</a> can be used for increased flexibility. When using a Server SDK in combination with the proper <a href="/docs/keys#scopes">API key scopes</a>, you can have any type of access to any of your project resources regardless of their permissions.</p>
+<p>Developers can use server integrations for increased flexibility. When using a Server SDK in combination with the proper <a href="/docs/keys#scopes">API key scopes</a>, you can have any type of access to any of your project resources regardless of their permissions.</p>
```
Comment (eldadfux) [inline app/views/docs/permissions.phtml:25]: Starting with "Developers" sounds a bit off, cold or remote. Can we start with "Server integration can be used for". We are all developers, it's not a 3rd party.

File: app/views/docs/admin.phtml [hunk 3]
```diff
@@ -1,7 +1,9 @@
-<p>The Appwrite API has two different modes it can use, the first mode is the client mode, which is the <b>default</b> mode, and the second one is the <b>admin</b> or server mode.</p>
+<p>You can use Appwrite to build both client and server integrations. Client integrations are for frontend and mobile applications, which use Appwrite's Client APIs and <a href="/docs/sdks#client">Client SDKs</a>. Server integrations are for backend and CLI applications, which use Appwrite's Server APIs and <a href="/docs/sdks#server">Server SDKs.</a></p>
```
Comment (eldadfux) [inline app/views/docs/admin.phtml:1]: Should we also mention the Appwrite CLI here?

File: app/views/docs/admin.phtml [hunk 4]
```diff
@@ -1,7 +1,9 @@
...
 
-<p>When using Appwrite from the client-side, you should go with the normal default mode. This mode allows every user of your project to access only resources they have been granted access to. When running in admin mode, you remove Appwrite default access restriction and ultimately allow access to any of the resources available on your Appwrite project (files, documents, or collections).</p>
+<p>When building client integrations with the Client SDKs, you authenticate with an account and operate in the scope of the account. This means Client SDKs can only access resources that your account has been granted permission to access.</p>
 
-<p>For security reasons, the admin mode only works in combination with an API key. You can create an API key from the Appwrite console, and you can choose what scopes of access you are willing to grant your SDK.</p>
+<p><a href="/docs/permissions"><i class="icon-angle-circled-right"></i>Learn more about permissions</a></p>
```
Comment (eldadfux) [inline app/views/docs/admin.phtml:5]: The icon problem here as well.

**Review by eldadfux — CHANGES_REQUESTED** [review]
Looks good, left a few comments.

**Comment by gewenyu99** [comment]
Screen Shot 2022-10-24 at 11 55 14 AM
@eldadfux This is also how it's done for the SDK links 🤔 Let me play around.

**Comment by gewenyu99** [comment]
Screen Shot 2022-10-24 at 1 36 27 PM
@eldadfux Does this look better?

File: app/views/docs/admin.phtml (hunk 3 above)
Comment (gewenyu99) [reply app/views/docs/admin.phtml]: Sure

File: app/views/docs/permissions.phtml (hunk 2 above)
Comment (gewenyu99) [reply app/views/docs/permissions.phtml]: Sure. I've made the change


**The conversation below is off topic**

I think we should consider if the docs should be written in a passive or active voice. There's a general distinction in grammar where:

> Server integrations can be used for increased flexibility.
This is a passive voice, which means there's no clear subject (persons) that we're addressing, which is generally considered weaker in writing, but acceptable for stuff like documentation. This is stating facts, but not suggesting you do something.

> You can use server integrations for increased flexibility.
This is considered an active voice. It's more instructive like you "should" do xxx or yyy to accomplish zzz. It's considered easier to read (at least what I was taught).

We should actually have a conversation about active vs passive voices in copy writing and docs... but that's a long conversation for later.

**Comment by eldadfux** [comment]
@gewenyu99 let's add a bit more spacing here, we can use on of our predefined `margin-bottom...` classes.
image

**Comment by gewenyu99** [comment]
Screen Shot 2022-10-26 at 12 26 51 PM
@eldadfux Changed margin to small instead of tiny

**Comment by gewenyu99** [comment]
With `margin-bottom` instead of `margin-bottom-small`.

**Review by eldadfux — APPROVED** [review]

**Pull request merged**